editor/plugins/animation_player_editor_plugin.cpp

```diff
@@ -974,6 +966,19 @@ void AnimationPlayerEditor::_update_player() {
 	_update_animation();
 }
 
+void AnimationPlayerEditor::_set_controls_disabled(bool p_disabled) {
+	stop->set_disabled(p_disabled);
+	play->set_disabled(p_disabled);
+	play_bw->set_disabled(p_disabled);
+	play_bw_from->set_disabled(p_disabled);
+	play_from->set_disabled(p_disabled);
+	frame->set_editable(!p_disabled);
```

Comment:
To make it easier to read I'd put this one above all the others, it's easy to miss that it's different if refactoring